raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -1593,6 +1644,13 @@ private Optional<Boolean> maybeHandleCommonResponse(
         int epoch,
         long currentTimeMs
     ) {
+        Optional<Node> leader = Optional.empty();
```

Comment:
I find that suggestion much harder to read because the important characters `?` and `:` get lost with all the other characters. I updated the code anyways.